client/package.json

```diff
@@ -88,6 +89,11 @@
     ],
     "commands": [
       {
+        "command": "extension.puppetShowConnectionMenu",
+        "category": "Puppet",
+        "title": "Show Connection Menu"
```

Comment:
Should we have the command show the menu? or should we just show the individual commands and the context menu just shows a curated list?